.github/workflows/docker-unified.yml

```diff
@@ -31,6 +31,9 @@ env:
   DATAHUB_ELASTIC_SETUP_IMAGE: "linkedin/datahub-elasticsearch-setup"
   DATAHUB_MYSQL_SETUP_IMAGE: "acryldata/datahub-mysql-setup"
   DATAHUB_UPGRADE_IMAGE: "acryldata/datahub-upgrade"
+  DATAHUB_INGESTION_BASE_IMAGE: "acryldata/datahub-ingestion-base"
+  DATAHUB_INGESTION_IMAGE: "acryldata/datahub-ingestion"
+  DATAHUB_INGESTION_SLIM_IMAGE: "acryldata/datahub-ingestion-slim"
```

Comment:
Trying to follow everything:

- slim depends on base
- main depends on slim

And in the definitions:
- base does the basic pip install base-requirements
- slim seems to do `pip install .[base]` followed by `pip install .[all]`
- main also seems do `pip install .[all]`

So I'm confused by that last step - what's the difference between slim and main?

Reply:
base -> slim -> main

Base: removed pyspark + deequ
Slim: *--no-deps* `pip install .[all]` (this prevents the excluded deps from getting pulled back)
Main: normal install including excluded deps

Its a slight difference but makes a difference in the image sizes. Not as great as I'd like but just over 1GB in space recovered.

Reply:
on slim: `pip install --no-deps .[all]` is a no-op, since `.` was already installed and the `--no-deps` tells pip to ignore the `[all]` directive

Reply:
Will pull that command then, thanks!